src/hoc18/Program2.java

```diff
@@ -0,0 +1,26 @@
+package hoc18;
+
+public class Program2 {
+	public static void main(String[] args) {
+		// take input
+		String input = "aWk60#ase2> o_0^4alis5L";
+		String[] inputArr = input.split(" ");
+
+		for (int i = 0; i < inputArr.length; i++) {
+			String element = inputArr[i];
+			char[] charArr = element.toCharArray();
+			int value = 0;
+			for (int j = 0; j < charArr.length; j++) {
+				int asciNumber = charArr[j];
+				if (asciNumber >= 48 && asciNumber <= 55) {
```

Comment:
```suggestion
				if (asciNumber >= 48 && asciNumber <= 57) {
```
ASCII values for digits ranges from 48 (=0) to 57(=9). In your program you are checking only upto 55. Effectively digits 8 and 9 present in the string would be ignored!